systems.csv

```diff
@@ -644,7 +644,7 @@ SI,Nomago Bikes - SLOVENIA,"Kranjska Gora, SI",nextbike_cx,https://bikes.nomago.
 SI,Nomago Bikes - ZANAPREJ,"Zagorje ob Savi, SI",nextbike_cf,https://bikes.nomago.si/,https://gbfs.nextbike.net/maps/gbfs/v2/nextbike_cf/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fgbfs.nextbike.net%2Fmaps%2Fgbfs%2Fv2%2Fnextbike_cf%2Fgbfs.json
 SK,Arriva Nitra Slovakia,"Nitra, SK",nextbike_as,https://arriva.bike/,https://gbfs.nextbike.net/maps/gbfs/v2/nextbike_as/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fgbfs.nextbike.net%2Fmaps%2Fgbfs%2Fv2%2Fnextbike_as%2Fgbfs.json
 SK,BikeKIA,"Žilina, SK",nextbike_ak,https://bikekia.sk/,https://gbfs.nextbike.net/maps/gbfs/v2/nextbike_ak/gbfs.json,https://gbfs-validator.netlify.app/?url=https%3A%2F%2Fgbfs.nextbike.net%2Fmaps%2Fgbfs%2Fv2%2Fnextbike_ak%2Fgbfs.json
-TR,AAR Bike,"Konya, TR",linka_aar_bike,https://aarbike.com.tr/,https://app.linkalock.com/api/gbfs/D2uRLGw8vBa2FJxa8/…,https://gbfs-validator.netlify.app/validator?url=https%3A%2F%2Fapp.linkalock.com%2Fapi%2Fgbfs%2FD2uRLGw8vBa2FJxa8%2F%E2%80%A6
+TR,AAR Bike,"Konya, TR",linka_aar_bike,https://aarbike.com.tr/,https://app.linkalock.com/api/gbfs/D2uRLGw8vBa2FJxa8/gbfs.json,https://gbfs-validator.netlify.app/validator?url=https%3A%2F%2Fapp.linkalock.com%2Fapi%2Fgbfs%2FD2uRLGw8vBa2FJxa8%2F%E2%80%A6
```

Comment:
Hello @drewda, 
Great catch! Thank you very much for this contribution.
Let's please update the validation report URL too.
Thank you!

```suggestion
TR,AAR Bike,"Konya, TR",linka_aar_bike,https://aarbike.com.tr/,https://app.linkalock.com/api/gbfs/D2uRLGw8vBa2FJxa8/gbfs.json,https://gbfs-validator.netlify.app/validator?url=https%3A%2F%2Fapp.linkalock.com%2Fapi%2Fgbfs%2FD2uRLGw8vBa2FJxa8%2Fgbfs.json
```